Bottom: 747e1307f82d376bbce8fd0ba8bdc056ed75bbb5
Top:    6ed2330c6be0021ae618c4411be02056c939009f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:19:07 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
